### PR TITLE
Users can see upcoming trips

### PR DESCRIPTION
**Pull request by kaylawoodbury**
### User Story
```
As a resident 
In order view upcoming shopping trips and times
I would like to be able to see who and when will be shopping
```
## Changes proposed in this pull request:
* Add screen stack for multiple 'pages'
* Add add view index of all active trips
* Move New Trip to Trips screen
* Add styling/formatting
* Update tests
* Refactored App.js file

## What I have learned working on this feature:
*  How to implement multi page apps using screen stack
*  How to style with React Native
*  How to do multi function onPress events
*  How to change the viewport size in cypress
* ....Basically how to do everything we know how to do in react in react native

## Packages/Gems added
- react-native-reanimated
## Screenshots (Client)
Screenshot 2020-04-15 at 14 50 39
Screenshot 2020-04-15 at 14 50 57
Screenshot 2020-04-15 at 14 51 21
Screenshot 2020-04-15 at 14 51 49
Screenshot 2020-04-15 at 14 52 09

## Lo-Fi's (Client)
Pasted Image at 2020_04_11_01_47 pm

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -1,47 +1,36 @@
...
-import NewTripForm from "./screens/NewTripForm";
 import axios from "axios";
-import LoginForm from "./screens/LoginForm";
-import { LinearGradient } from "expo-linear-gradient";
+import TripsList from "./screens/TripsList";
+import { NavigationContainer } from "@react-navigation/native";
+import { createStackNavigator } from "@react-navigation/stack";
 
-// axios.default.baseURL = "http://localhost:3000";
+//  axios.default.baseURL = "http://localhost:3000";
```
Comment (emtalen) [inline App.js]: remove comments

File: modules/tripActions.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { GET_TRIP_LIST } from "../state/actions/actionTypes";
+
+const fetchTrips = async (dispatch) => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  let response = await axios.get("http://localhost:3000/pings", {
+    headers: headers,
```
Comment (emtalen) [inline modules/tripActions.js:7]: change to deployed link

File: modules/tripActions.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+import { GET_TRIP_LIST } from "../state/actions/actionTypes";
+
+const fetchTrips = async (dispatch) => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  let response = await axios.get("http://localhost:3000/pings", {
```
Comment (pgaunitz) [inline modules/tripActions.js]: change to heruko

**Review by emtalen — APPROVED** [review]
Just think about the links but really good job on this one! 💯

**Review by pgaunitz — COMMENTED** [review]
LDGTM

**Review by pgaunitz — APPROVED** [review]

**Pull request merged**